vendor/assets/javascripts/neat/renderer/basic.coffee

```diff
@@ -41,6 +43,9 @@ class window.Neat.Renderer.Basic
         @_moveView(viewIndex, index) unless viewIndex is index
       else
         @_insertView @view.buildViewFor(model), index
+
+    if @afterRenderObservers
+      callback() for callback in @afterRenderObservers
```

Comment:
Rather than implementing a quick observer pattern, maybe use the already-included [observer.js](https://github.com/boblail/neat-rails/blob/0.5/vendor/assets/javascripts/neat/lib/observer.js) — or, better yet, [Backbone.Events](http://backbonejs.org/#Events). I think we might be able to drop `observer.js` in favor of `Backbone.Events` which'll reduce dependencies.